### PR TITLE
Suggestions

### PR DESCRIPTION
**Pull request by alichtman**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,23 +6,22 @@
...
 
 Besides autotrading, BitVision's key features are:
 
-- Real-time monitoring of Bitcoin-related news, technical indicators, and blockchain data (i.e. hash rate, avg. block size, etc.)
+- Real-time monitoring of Bitcoin-related news, technical indicators, and blockchain network data.
 - Log of previous transactions and current account balance
 - Portfolio metrics, including your sharpe ratio, buy and sell accuracy, net profit, and returns
 - Easy toggling of autotrading and ability to manually make trades
-
-Unlike other systems, there's no need to run BitVision on localhost or host a database – simply install and run `$ bitvision` to get started.
```
Comment (alichtman) [inline README.md]: This is really a feature, and the other part is pretty overlapping with the next section.

File: README.md [hunk 2]
```diff
@@ -6,23 +6,22 @@
...
+- Unlike other systems, there's no need to run BitVision on localhost or host a database.
 
 ## Usage
 
-> Requires Node v10+ and Python 3.
+> Requires `Node v10+` and `Python 3`.
```
Comment (alichtman) [inline README.md]: It looks nice, okay

File: README.md [hunk 3]
```diff
@@ -63,7 +62,7 @@ Who ever said it works?
 
 ## Authors
 
-The BitVision "frontend" was built by [@alichtman](http://github.com/alichtman), and runs on the Blessed.js library. The "backend" was built by [@shobrook](http://github.com/shobrook), and runs on the SciPy stack.
+The BitVision "frontend" was built by [@alichtman](http://github.com/alichtman), and runs on the `Blessed.js` library. The "backend" was built by [@shobrook](http://github.com/shobrook), and runs on the `SciPy` stack.
```
Comment (alichtman) [inline README.md]: Like, really nice.

**Comment by shobrook** [comment]
Thanks Mr. Shobrook!

**Pull request merged**